ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -192,11 +186,11 @@ def test_concurrent_server_spans(self):
 
         # Capture the currently active span in each thread
         active_spans_in_handler = []
-        latch = get_latch(2)
+        lock = threading.Lock()
 
         def handler(request, context):
-            latch()
-            active_spans_in_handler.append(tracer.get_current_span())
+            with lock:
```

Comment:
The point of `latch` is to make sure both handlers are active at once, to simulate handling two simultaneous gRPC requests. With the lock this test does the same thing as `test_sequential_server_spans`.